src/linter/ui5Types/SourceFileLinter.ts

```diff
@@ -35,6 +37,12 @@ export default class SourceFileLinter {
 	// eslint-disable-next-line @typescript-eslint/require-await
 	async lint() {
 		try {
+			const dataTypes = await fs.readFile(
```

Comment:
Is it really necessary to read and parse this file every time a file is linted?

Reply:
Moved it out, so it could be read only once per linter execution